docs/src/examples/components/Button/Usage/index.tsx

```diff
@@ -9,6 +10,16 @@ const Usage = () => (
       description='A button used in cards is a "tinted" version of a default button.'
       examplePath="components/Button/Usage/ButtonUsageExample"
     />
+    <ComponentExample
+      title="With tooltip"
+      description={
+        <>
+          {'The button as actionable element should be rendered with '}
```

Comment:
sorry, cannot understand what is the main idea this description is going to convey to the client?